### PR TITLE
brutal cleanup with comments cleaning

### PR DESCRIPTION
**Pull request by YUzhva**
### JIRA:
**LR-167 Cleanup**: https://ssu-jira.softserveinc.com/browse/LR-167
subtask: **LR-165 Remove all unnecessary yml files**: https://ssu-jira.softserveinc.com/browse/LR-165

### Description:
Main purpose of this LR-167 task - to clean our application from empty/old files/folders and files that we never used before.

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Code reviewers for this pull request:

> @anatoliiibrahimov 
> @brouhaha88 
> @sergiy17 

- [ ] @OKrain 
- [ ] @superNazo 
- [x] @vovka 

### Fix summary. 

Deleted:
- all unnecessary filles (including .keep files)
- all unnecessary folders
- all comments from existing files that wasn't generated by rails

### Testing approach.
Manual testing (rails s) application still works :fire:

### CODE REVIEW

File: app/controllers/video_recorder_controller.rb [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-
-  def ziggeo_init
-    @ziggeo = Ziggeo.new(Rails.application.secrets.ziggeo_api_token,
-                         Rails.application.secrets.ziggeo_private_key,
-                         Rails.application.secrets.ziggeo_encryption_key)
-  end
-
-  def all_videos
-    @all_videos = []
-    @ziggeo.videos.index.map { |video| @all_videos.push(video["token"]) }
```
Comment (superNazo) [inline app/controllers/video_recorder_controller.rb]: will recorder work without it? or it will be rewritten on angular?

Comment (YUzhva) [inline app/controllers/video_recorder_controller.rb]: you are right - recorder wouldn't work without that controller.

If we don't need separate page for testing those video recorder, we could move this two functions  ziggeo_init and all_videos to the controller, that would work with videocontroller, besides we don't need to take our videoi tokens for player from their service, we are saving video_token in answers model, so we could send them in .json

Comment (superNazo) [inline app/controllers/video_recorder_controller.rb]: then, everything's fine, I think

Comment (OKrain) [inline app/controllers/video_recorder_controller.rb]: looks good if everything still works

**Comment by vovka** [comment]
OK, let's merge this!:)

**Comment by vovka** [comment]
Despite there are no conflicts and branch is mergable, there are 9 commits though. Please, rebase it to include only valuable commit(s).

**Comment by vovka** [comment]
:ok:

**Comment by vovka** [comment]
This PR had been reviewed and approved, hadn't it? @YUzhva what's the status of this?

**Comment by vovka** [comment]
@YUzhva ^^^

**Comment by YUzhva** [comment]
@vovka after last --rebase with master is stops to work (=

File: app/assets/javascripts/quizzes_controller.js [hunk 2]
```diff
@@ -18,7 +18,9 @@ QuestionField.prototype.bindCocoonEvents = function() {
 };
 
 $(document).on("ready page:load", function() {
-  var object = new QuestionField($("#questions"), function() { return $("#questions .question .nested-fields"); }, $("#add_question"));
+  var object = new QuestionField($("#questions"), function() {
+    return $("#questions .question .nested-fields");
+  }, $("#add_question"));
```
Comment (vovka) [inline app/assets/javascripts/quizzes_controller.js]: Codacy Issue found: ["$" is not defined.](https://www.codacy.com/app/vovka/video_quiz/file/2602465272/issues/source?bid=3205144&fileBranchId=3242172#l23)

File: app/assets/javascripts/angularjs_app/components/home/homeControllers.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+var homeControllers = angular.module('homeControllers', []);
```
Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/home/homeControllers.js]: Codacy Issue found: ["angular" is not defined.](https://www.codacy.com/app/vovka/video_quiz/file/2602465266/issues/source?bid=3205144&fileBranchId=3242172#l1)

Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/home/homeControllers.js]: Codacy Issue found: ["angular" is not defined.](https://www.codacy.com/app/vovka/video_quiz/file/2602697743/issues/source?bid=3205144&fileBranchId=3242172#l1)

**Comment by vovka** [comment]
:ok

File: app/assets/javascripts/angularjs_app/components/home/homeControllers.js (hunk 3 above)
Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/home/homeControllers.js]: Codacy Issue found: ["angular" is not defined.](https://www.codacy.com/app/vovka/video_quiz/file/2605271475/issues/source?bid=3205144&fileBranchId=3242172#l1)

**Pull request merged**